engine/render/src/render/font.cpp

```diff
@@ -96,6 +96,7 @@ namespace dmRender
 
         font_map->m_CellTempData = (uint8_t*)malloc(font_map->m_CacheCellWidth*font_map->m_CacheCellHeight*4);
 
+        font_map->m_CacheCursor = 0;
```

Comment:
Another bug fix (unreported, as noone has really used the extension functionality)